### ISSUE TITLE
Interference with right click context menus

### ISSUE THREAD

**Issue by luispabon** [overdodactyl/ShadowFox#264]
Seems to happen on any context menu, including on the tab bar:

![image](https://user-images.githubusercontent.com/6388823/57987585-4b09a480-7a7b-11e9-91af-5d91325dde56.png)

instead of:

**Comment by githubtefo** [comment]
Yes, same here. Is there any way to customize this and avoid changes in the context menus?

**Comment by evertonstz** [comment]
Same problem here. Seems to be linux related.

**Comment by githubtefo** [comment]
Any updates on this regard? Thanks!

**Comment by dhasial** [comment]
This is still an issue. I'm doing some modification of the userChrome.css to see if I can pinpoint exactly what is causing this, but it would be helpful to have the input of someone who wrote the theme. It's the worst to have such a nice theme marred by something that affects every single context menu.

The very fact that removing the custom CSS fixes this means there's almost definitely a specific setting or group of settings that is causing this.
